packages/builder/src/components/start/ExportAppModal.svelte

```diff
@@ -16,7 +16,7 @@
 
   let password = null
   const validation = createValidationStore()
-  validation.addValidatorType("password", "password", true)
+  validation.addValidatorType("password", "password", true, { minLength: 8 })
```

Comment:
do we need a min length?

Reply:
This min length was actually added for the encrypted exports. It was no other referecne to this validator type, so I assumed that it was not used. But it's used dynamically based on datasource schemas, so my change was adding a min length requirement to all the passwords. I only wanted it for the exporting encryption